tests/test_database.py

```diff
@@ -1,5 +1,3 @@
-import pytest
```

Comment:
Aren't this necessary?

Reply:
Not at all, even pylint suggests that it is redundant